tests/integration/test_attachments.py

```diff
@@ -20,28 +21,32 @@ def test_attachments_400(mini_sentry, relay_with_processing, attachments_consume
     assert excinfo.value.response.status_code == 400
 
 
-def test_attachments_with_processing(
+def test_mixed_attachments_with_processing(
```

Comment:
I feel like we should also have an integration test for an error with an attachment in the same envelope and a transaction with an attachment in the same envelope, just to make sure that those cases did not regress.

Reply:
done